CMakeLists.txt

```diff
@@ -68,6 +68,10 @@ set(TARGET_MKDISP "mkdisp")
 # Defined in src/common/CMakeLists.txt via command add_library
 set(TARGET_LIBCOMMON_STATIC "common")
 
+# Generates object file (shared library) `libsleefdft`
+# Defined in src/dft/CMakeLists.txt via command add_library
+set(TARGET_LIBDFT "sleefdft_shared")
```

Comment:
I think the name `sleefdft` is enough, no need to append `_shared`. As you can see from https://travis-ci.org/shibatch/sleef/jobs/309529448#L1158, the resulting library has a .so extension that makes the fact that the library is a shared object explicit.